Source/missiles.cpp

```diff
@@ -3971,9 +3971,7 @@ void MI_Immolation(Sint32 i)
 		int v = 2 * missile[i]._miVar6;
 		missile[i]._miVar6 = v;
 		missile[i]._miVar7 = v;
-		missile[i]._mimfnum--;
-		if (missile[i]._mimfnum < 0)
-			missile[i]._mimfnum = 7;
+		missile[i]._mimfnum = left[missile[i]._mimfnum];

```

Comment:
_mimfnum also states it's a direction:
`Sint32 _mimfnum; // The direction of the missile (direction enum)`
should it also be updated? Or is this something for another clean-up pr?

Reply:
see the PR description, _mimfnum is used for multiple things.